pom.xml

```
@@ -43,6 +43,7 @@
         <jcommander.version>1.35</jcommander.version>
         <log4j.version>1.2.17</log4j.version>
         <jackson.version>2.9.9</jackson.version>
+        <jackson-databind.version>2.9.9.2</jackson-databind.version>
```

```suggestion
        <jackson-databind.version>2.9.9.3</jackson-databind.version>
```
Not sure at all we're affected by this, but `2.9.9.2` introduced a regression that was fixed today in `2.9.9.3` according to [this](https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.9#micro-patches), maybe we can include it as well while we're updating?

---

nice find, let me update this then (here's hoping there won't be yet another micro release in a few days 😅 )

---

done, can you review again @KSerrania?